proto/dymension/eibc/query.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+package dymensionxyz.dymension.eibc;
+
+import "gogoproto/gogo.proto";
+import "google/api/annotations.proto";
+import "cosmos/base/query/v1beta1/pagination.proto";
+import "dymension/eibc/params.proto";
+
+option go_package = "github.com/dymensionxyz/dymension/v3/x/eibc/types";
+
+// Query defines the gRPC querier service.
+service Query {
```

Comment:
what about querying avaialable DemandOrders?

Reply:
gonna do through the indexer services using the events. don't want to expose it directly to RPC. 
as I see it as a rule of thumb the blockchain node should only be queried for validation/proofs and not filtering/searches that can be done using indexers. prefer not to expose this api.